### PR TITLE
Updated latvian translation

### PR DESCRIPTION
**Pull request by CanPotate**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...
- Related #issue...

**Details of Changes**
Add details of changes here.
- Added...
- Fixed...

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: Files/MultilingualResources/Files.lv-LV.xlf [hunk 1]
```diff
@@ -2852,607 +2848,607 @@
...
-          <target state="new">Enter compact overlay</target>
+          <target state="translated">Atvērt kompakto pārklāju</target>
         </trans-unit>
         <trans-unit id="NavigationToolbarExitCompactOverlay.Label" translate="yes" xml:space="preserve">
           <source>Exit compact overlay</source>
-          <target state="new">Exit compact overlay</target>
+          <target state="translated">Iziet kompakto pārklājumu</target>
         </trans-unit>
-        <trans-unit id="WSL" translate="yes" xml:space="preserve">
+        <trans-unit id="WSL" translate="no" xml:space="preserve">
```
Comment (BanCrash) [inline Files/MultilingualResources/Files.lv-LV.xlf:3181]: Is there any reason to change this to "no"?

**Review by winston-de — APPROVED** [review]

File: Files/MultilingualResources/Files.lv-LV.xlf (hunk 1 above)
Comment (yaichenbaum) [reply Files/MultilingualResources/Files.lv-LV.xlf:3181]: Maybe I'm wrong but I would assume WSL is the same in all languages.

**Comment by yaichenbaum** [comment]
@CanPotate thank you!

File: Files/MultilingualResources/Files.lv-LV.xlf (hunk 1 above)
Comment (BanCrash) [reply Files/MultilingualResources/Files.lv-LV.xlf:3181]: So "final" state could be used when the translation is the same as the original. Gotcha!

**Pull request merged**